apps/ftu/test/unit/app_test.js

```diff
@@ -1,78 +0,0 @@
-/* global MocksHelper, MockNavigatorMozMobileConnections, MockL10n,
```

Comment:
its a bit crazy that the only test for app.js is a collection one. But I can put this back easily enough with new coverage so I guess removing it for now is the right thing to do for this patch